drivers/bluetooth/functionJVM.go

```diff
@@ -0,0 +1,19 @@
+//go:build android
+
+package bluetooth
+
+import (
+	"errors"
+	"fyne.io/fyne/v2/driver"
+)
+
+var runOnJVM = originForm
+
+func originForm(fn func(vm, env, ctx uintptr) error) error {
```

Comment:
do you agree with thar?

Reply:
I don't understand the question. (Or maybe I just don't understand the name "originForm").
Using build tags we can switch in different "runNative" functions so the type assertion in your code is safe.

Reply:
simply I do not consider clean everywhere write cast construction ..., so question is whether I must rewrite it to all function to this form or I can keep this solution (maybe "originForm" rename to "unwrap" or "uncast" ... or the most simply  use `//go:link fyne.io/fyne/internal/.../mobileinit/RunOnJVM` - next question is which one you what you prefer)

Reply:
I don't understand what then indirection is trying to achieve? Just call the method "runNative" or similar, and then have a no-op version in a non-android file. Global variables and go:link just make this more complex font they?

Reply:
ok, go:linkname I will not use ...